src/app/rundown-view/services/mini-shelf-state.service.spec.ts

```diff
@@ -0,0 +1,237 @@
+import { MiniShelfStateService } from './mini-shelf-state.service'
+import { anyString, anything, instance, mock, when } from '@typestrong/ts-mockito'
+import { RundownEventObserver } from '../../core/services/rundown-event-observer.service'
+import { ActionService } from '../../shared/abstractions/action.service'
+import { Rundown } from '../../core/models/rundown'
+import { Segment } from '../../core/models/segment'
+import { Tv2SegmentMetadata } from '../../core/models/tv2-segment-metadata'
+import { Tv2ActionContentType, Tv2VideoClipAction } from '../../shared/models/tv2-action'
+import { Observable } from 'rxjs'
+import { PartSetAsNextEvent, PartTakenEvent } from '../../core/models/rundown-event'
+
+describe(MiniShelfStateService.name, (): void => {
+  it('should create', (): void => {
+    const testee: MiniShelfStateService = createMinimalTestee()
+    expect(testee).toBeTruthy()
+  })
+
+  describe(MiniShelfStateService.prototype.setActions.name, (): void => {
+    it('should be defined', (): void => {
+      const testee: MiniShelfStateService = createMinimalTestee()
+      expect(testee.setActions).toBeDefined()
+    })
```

Comment:
This test is redundant (and all the other like it).
We are in Typescript. If the method doesn't exist we are going to compile errors all the places we use it.